build.sh

```diff
@@ -5,6 +5,7 @@ python -m pip install --upgrade pip
 python -m pip install --upgrade pylint
 if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 python -m pip install --upgrade Django
+python -m pip install --upgrade mysqlclient
```

Comment:
Add "|| { echo "Mysql client install failed"; exit 1; }" to the end of this line, see #19 changes if needed.